src/app/sections/GraphicBody.svelte

```diff
@@ -126,6 +125,8 @@ function handleBodySelectors(event) {
   background-color: white;
   padding: var(--space-4x);
   padding-top: var(--space-2x);
+  border-radius: var(--border-radius-1h);
+  box-shadow: 0 0 3px 2px rgba(0,0,0,0.1) inset;
```

Comment:
```suggestion
  border: 1px solid var(--cool-gray-100);
```

Reply:
The inset shadow is kind of intense and might be a bit distracting. I'd maybe recommend:

1. removing that,
2. adding a 1px solid border to the body element